docs(VNavigationDrawer): add difference between v-model:rail and rail="true" props (#17003)
vuetifyjs/vuetify: packages/docs/src/pages/en/components/navigation-drawers.md

<alert type="info">

>   The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to drawer expansion, instead of **expand-on-hover**, you can add create a data prop `rail` and then add this to your nav bar `:rail="rail"` followed by `@mouseover="rail = false"` and `@mouseleave="rail = true"`.

Reviewer: ```suggestion
  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to **expand-on-hover**, bind **v-model:rail** to a data prop.
```

>   The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to drawer expansion, instead of **expand-on-hover**, you can add create a data prop `rail` and then add this to your nav bar `:rail="rail"` followed by `@mouseover="rail = false"` and `@mouseleave="rail = true"`.

Reviewer 2: ```suggestion
  The **expand-on-hover** prop does not alter the content area of **v-main**. To have content area respond to **expand-on-hover**, bind **v-model:rail** to a data prop.
```
